image_transport/src/publisher.cpp

```diff
@@ -202,6 +202,37 @@ void Publisher::publish(const sensor_msgs::msg::Image::ConstSharedPtr & message)
   }
 }
 
+void Publisher::publish(sensor_msgs::msg::Image::UniquePtr message) const
+{
+  if (!impl_ || !impl_->isValid()) {
+    // TODO(ros2) Switch to RCUTILS_ASSERT when ros2/rcutils#112 is merged
```

Comment:
This PR https://github.com/ros2/rcutils/pull/112 was closed, do you mind to remove this comment ?